ledger/catchpointtracker_test.go

```diff
@@ -592,7 +597,12 @@ func TestCatchpointReproducibleLabels(t *testing.T) {
 		ml2 := ledgerHistory[rnd]
 		require.NotNil(t, ml2)
 
-		ct2 := newCatchpointTracker(t, ml2, cfg, ".")
+		cfg2 := cfg
+		// every other iteration modify CatchpointTracking to ensure labels generation does not depends on catchpoint file creation
+		if rnd%2 == 0 {
+			cfg2.CatchpointTracking = int64(crypto.RandUint63())%2 + 1 //values 1 or 2
+		}
```

Comment:
this is not guaranteed to modify the tracking value, for what looks like 2 reasons:

1) the 50% chance assignment, depending on the duration of this loop, might fail to modify the value (rare but flakey behavior)

2) the loop modifies this `rnd` value by `rnd -= cfg.CatchpointInterval`, which in this test appears to be 50. So I would expect `rnd%2` to always be true/false for a given test

why not just unconditionally set `CatchpointTracking = (CatchpointTracking % 2) + 1` on every loop?